### PR TITLE
Add eslint and prettier

### PR DESCRIPTION
**Pull request by bazelajoanna**
ESlint and prettier has been added based on documentation https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
+dist
+# don't lint nyc coverage output
+coverage
```
Comment (franiakat) [inline .eslintignore]: We don't have either `coverage` or `dist` folder - build is created in `build` directory, so we can change `dist` to `build`.

**Review by franiakat — CHANGES_REQUESTED** [review]
Just change the `.eslintignore` and you are free to go. :)

File: .eslintignore (hunk 1 above)
Comment (bazelajoanna) [reply .eslintignore]: Ok, done.

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
+build
+# don't lint nyc coverage output
+coverage
```
Comment (franiakat) [inline .eslintignore]: Coverage can be removed, as we don't have the coverage setup (probably referring to test coverage count) ;)

Comment (bazelajoanna) [reply .eslintignore]: Removed :)

**Review by franiakat — APPROVED** [review]
Good to go, please use Squash and Merge option. 🎉

**Pull request merged**